ide-style/intellij/sofi-intellij-code-style.xml

```diff
@@ -23,6 +23,17 @@
       <emptyLine />
       <package name="com.sofi" withSubpackages="true" static="false" />
       <emptyLine />
+      <package name="configs" withSubpackages="true" static="false" />
```

Comment:
`configs` , `controllers`, `exceptions` and `models` are all grouped together?

Reply:
(shrug) They are usually all related to Play classes, so I left them grouped.

Reply:
But I mean, why are `services` and `views` excluded from that grouping? Don't they also fall into the category of package roots that are usually uniquely provided by play apps for their own logic?

Reply:
I looked some existing codebases, and most of them had services and views broken out with a newline, so this tended to cause the least amount of disturbance in the existing projects.

If you think the addition or removal of emptyLine is of benefit, I have no opinion on adding/removing them.